conf/kaust.config

```diff
@@ -0,0 +1,71 @@
+// KAUST Config Profile
+params {
+    config_profile_name = 'KAUST'
+    config_profile_description = 'Profile for use on King Abdullah Univesity of Science and Technology (KAUST) Ibex Cluster.'
+    config_profile_contact = 'Husen Umer (@kaust.edu.sa)'
+    config_profile_url = 'https://docs.hpc.kaust.edu.sa/quickstart/ibex.html'
+    save_reference = false
+    igenomes_ignore = true
+}
+
+// Load genome resources and assets hosted by the Bioinformatics team on IBEX cluster
+includeConfig '/biocorelab/BIX/resources/configs/genomes.yaml'
+
+singularity {
+  enabled = true
+  autoMounts = true
+  pullTimeout = '60 min'
+  // Use existing images from the centralized library, if available
+  libraryDir = "/biocorelab/BIX/resources/singularity/images/"
+  // Download images that are missing from the library to user space
+  cacheDir = "/home/$USER/.singularity/nf_images/"
```

Comment:
Please double check this works and nextflow doesn't try to resolve the variable [rather than bash] (this can happen in some of the config sections)

Reply:
And does the cache directory already exist for all users?

Reply:
I have already tested this and it works :) 
yes, it will automatically create the cache directory if it doesn't exist